src/components/common/CompoundLayout/components/Root.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, useColorMode } from "@chakra-ui/react"
+import { PropsWithChildren } from "react"
+import { LAYOUT_MAX_WIDTH_CSS_VAR } from "../constants"
+
+type Props = PropsWithChildren<{
+  background?: string
+  maxWidth?: string
+}>
+
+export function Root({
+  children,
+  background,
+  maxWidth = "sizes.container.lg",
+}: Props) {
+  const { colorMode } = useColorMode()
+
+  return (
+    <Box
+      sx={{ [LAYOUT_MAX_WIDTH_CSS_VAR]: maxWidth }}
```

Comment:
I have a suggestion for an alternative solution here. What if we'd remove this constant & type the `maxWidth` prop as `ChakraProps["width"]`? That way we could just use `maxWidth="container.lg"` here. We could also store the default value in a constant inside this file instead of a new one & export it, that way we wouldn't really need the `callCssVariable` util.